src/app/components/Figure/index.jsx

```diff
@@ -2,10 +2,16 @@ import React from 'react';
 import { string } from 'prop-types';
 import styled from 'styled-components';
 import Caption from './Caption';
+import VisuallyHiddenText from '../VisuallyHiddenText';
 import { GEL_SPACING_DBL } from '../../lib/constants/styles';
 
 const renderCaption = caption =>
-  caption ? <Caption>{caption}</Caption> : null;
+  caption ? (
+    <Caption>
+      <VisuallyHiddenText>Image caption,</VisuallyHiddenText>
```

Comment:
Could you please store the text in a constant - this will make it easier to localise this component later. Also, you can then add a space after the `,` without the linter removing it automatically. :)